web/src/routes/admin/+layout.svelte

```diff
@@ -86,7 +86,7 @@
 			</div>
 
 			<section id="setting-content" class="flex place-content-center mx-2">
-				<section class="w-[800px] pt-5 pb-28">
+				<section class="w-full pt-5 pb-28">
```

Comment:
While fullwidth is good for mobile, you have to be careful with larger screens. Right now (on a 2560x1440 screen) all admin pages look stretched out with lots of empty space between elements. You can have a look at [using a container](https://tailwindcss.com/docs/container) or just define a [max width](https://tailwindcss.com/docs/max-width) either for the layout or individual pages.

Reply:
fixed in 7b77a64205feefae7e1c40a6f3b5b3a6a39fa150